mslib/msui/mscolab.py

```diff
@@ -928,9 +928,8 @@ def upload_image(self, _=None):
                                      f'Cannot identify image file. Please check the file format. Error: {e}')
 
     @verify_user_token
-    def delete_account(self, _=None):
-        # ToDo rename to delete_own_account
-        reply = QMessageBox.question(
+    def delete_own_account(self, _=None):
+         reply = QMessageBox.question(
```

Comment:
```suggestion
        reply = QMessageBox.question(
```

This space breaks linting and tests right now.